library/swift/ios/UserDefaultsStore.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+/// Simple implementation of a `KeyValueStore` leveraging `UserDefaults` for persistence.
+@objcMembers
+public class UserDefaultsStore: KeyValueStore {
```

Comment:
There's no need for `@objcMembers` and since this class isn't useful to subclass, we should mark it as `final`:

```suggestion
public final class UserDefaultsStore: KeyValueStore {
```

Reply:
Isn't @objcMembers necessary to call this from Objective-C code?

Reply:
yes, you need `@objcMembers` at the class level or `@objc` at method level if you want this class and its methods to be usable from objc.

Reply:
> Isn't @objcMembers necessary to call this from Objective-C code?

No, this just stores properties (i.e. members) in the ObjC runtime, but we don't need to access the stored properties of this class at all.